lizmap/server/lizmap_filter.py

```diff
@@ -47,43 +47,67 @@ def __init__(self, server_iface: 'QgsServerInterface') -> None:
 
     def requestReady(self):
         try:
+            # Get QGIS Project path
             config_path = self.iface.configFilePath()
             if not os.path.exists(config_path):
+                # QGIS Project path does not exist as a file
+                # The request can be  evaluated by QGIS Server
                 return
 
+            # Get Lizmap config path
             config_path += '.cfg'
             if not os.path.exists(config_path):
+                # Lizmap config path does not exist
+                # The request can be  evaluated by QGIS Server
                 return
 
+            # Get Lizmap config
             cfg = None
             with open(config_path, 'r') as cfg_file:
                 try:
                     cfg = json.loads(cfg_file.read())
                 except Exception:
+                    # Lizmap config is not a valid JSON file
+                    # The request can be  evaluated by QGIS Server
                     return
 
             if not cfg:
+                # Lizmap config is empty
+                # The request can be  evaluated by QGIS Server
                 return
 
+            # Check project acl option
             cfg_options = cfg['options']
             if 'acl' not in cfg_options or not cfg_options['acl']:
+                # No acl defined
```

Comment:
Should log something, at least in debug mode.

Reply:
The config options can have no `acl`, do I have to log acl defined and acl not defined ?

Reply:
log message added